src/async_impl/client.rs

```diff
@@ -1406,6 +1432,32 @@ impl ClientBuilder {
         self
     }
 
+    /// Add a certificate revocation list.
+    ///
+    ///
+    /// # Optional
+    ///
+    /// This requires the `rustls-tls(-...)` Cargo feature enabled.
+    #[cfg(feature = "__rustls")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "rustls-tls")))]
+    pub fn add_crl(mut self, crl: Crl) -> ClientBuilder {
+        self.config.crls.push(crl);
+        self
+    }
+
+    /// Add multiple certificate revocation lists.
+    ///
+    ///
+    /// # Optional
+    ///
+    /// This requires the `rustls-tls(-...)` Cargo feature enabled.
+    #[cfg(feature = "__rustls")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "rustls-tls")))]
+    pub fn add_crls(mut self, mut crls: Vec<Crl>) -> ClientBuilder {
```

Comment:
Is there a need for this second method? Is having a vec of them common? In favor of leaving this method off, because otherwise there's an additional question of forcing it to be in a `Vec<_>`, or allowing a `&[_]`, or even just `impl IntoIterator`. If we don't start with the method, we can skip that decision :)

Reply:
It is a common practice to concatenate several PEM CRLs and put them to one file. Thus, the file will be read as a bundle. I think it will be more convinient to have a method to add them all via one call. I agree that the parameter type should be changed if we agree to leave the method.